bytecode/tests/test_cfg.py

```diff
@@ -429,5 +430,163 @@ def test_get_block_index(self):
         self.assertRaises(ValueError, blocks.get_block_index, other_block)
 
 
+class CFGStacksizeComputationTests(TestCase):
+
+    def check_stack_size(self, func):
+        code = func.__code__
+        bytecode = Bytecode.from_code(code)
+        cfg = ControlFlowGraph.from_bytecode(bytecode)
+        from bytecode import dump_bytecode
+        dump_bytecode(bytecode)
+        dump_bytecode(cfg)
+        assert code.co_stacksize == cfg.compute_stacksize()
```

Comment:
Please use assertEqual(..., ...). It's not removed when python is run with -O, and it provides better error message on test failure.